packages/adapters-library/src/adapters/morpho-blue/common/morphoBluePoolAdapter.ts

```diff
@@ -54,10 +56,15 @@ const morphoBlueContractAddresses: Partial<
   },
 }
 
-export abstract class MorphoBluePoolAdapter implements IMetadataBuilder {
+export abstract class MorphoBluePoolAdapter
+  implements IMetadataBuilder, IProtocolAdapter
```

Comment:
It was missing `IProtocolAdapter`, it's just a coincidence that all methods were implemented.